internal/firebase/error.go

```diff
@@ -36,6 +39,13 @@ func (err *ErrorDetails) Error() string {
 	return err.Err
 }
 
+func (err *ErrorDetails) Is(target error) bool {
```

Comment:
I don't think you should need this? It also doesn't guarantee that the error is an ErrorDetails because it could be wrapped.

Reply:
I have the static ones up top, but they don't match unless it's a complete match and I also specify the http code for them. I was hoping to match only on this error enum firebase gives back